doc/sphinx/source/inspection/profiling.rst

```diff
@@ -0,0 +1,6 @@
+.. inspection/profiling.rst:
+
+.. _profiling: 
+
+Profiling Performance
+#####################
```

Comment:
we should mention nbench here 
nbench is also good tool for debugging. We should have a section on nbench somewhere

Reply:
Good idea.  Added.